src/routes/AgreementViewRoute.js

```diff
@@ -128,7 +136,16 @@ class AgreementViewRoute extends React.Component {
       fetch: props => !!props.stripes.hasInterface('organizations-storage.interfaces', '1.0 2.0'),
     },
     interfaceRecord: {},
-    agreementEresourcesOffset: { initialValue: 0 },
+    agreementEresourcesConfig: {
+      initialValue: {
+        agreementEresourcesOffset:  0,
+      }
+    },
+    agreementLinesConfig: {
+      initialValue: {
+        agreementLinesOffset:  0,
+      }
+    },
```

Comment:
I dont think now that we need this additional `agreementEresourcesConfig` and `agreementLinesConfig` resources here. We had it because we wanted the `initialLoad` in there as well. You could directly work with `agreementEresourcesOffset` and `agreementLinesOffset`?